doc/build-netbsd.md

```diff
@@ -1,81 +1,145 @@
-NetBSD build guide
-======================
-(updated for NetBSD 8.0)
+# NetBSD Build Guide
 
-This guide describes how to build bitcoind and command-line utilities on NetBSD.
+**Updated for NetBSD [9.1](https://netbsd.org/releases/formal-9/NetBSD-9.1.html)**
 
-This guide does not contain instructions for building the GUI.
+This guide describes how to build bitcoind, command-line utilities, and GUI on NetBSD.
 
-Preparation
--------------
+## Dependencies
 
-You will need the following modules, which can be installed via pkgsrc or pkgin:
+The following dependencies are **required**:
+
+ Library                                          | Purpose    | Description
+ -------------------------------------------------|------------|----------------------
+ [autoconf](https://pkgsrc.se/devel/autoconf)     | Build      | Automatically configure software source code
+ [automake](https://pkgsrc.se/devel/automake)     | Build      | Generate makefile (requires autoconf)
+ [libtool](https://pkgsrc.se/devel/libtool)       | Build      | Shared library support
+ [pkg-config](https://pkgsrc.se/devel/pkg-config) | Build      | Configure compiler and linker flags
+ [git](https://pkgsrc.se/devel/git)               | Clone      | Version control system
+ [gmake](https://pkgsrc.se/devel/gmake)           | Compile    | Generate executables
+ [boost](https://pkgsrc.se/meta-pkgs/boost)       | Utility    | Library for threading, data structures, etc
+ [libevent](https://pkgsrc.se/devel/libevent)     | Networking | OS independent asynchronous networking
+
+
+The following dependencies are **optional**:
+
+  Library                                            | Purpose          | Description
+  ---------------------------------------------------|------------------|----------------------
+  [db4](https://pkgsrc.se/databases/db4)             | Berkeley DB      | Wallet storage (only needed when wallet enabled)
+  [qt5](https://pkgsrc.se/x11/qt5)                   | GUI              | GUI toolkit (only needed when GUI enabled)
+  [qrencode](https://pkgsrc.se/converters/qrencode)  | QR codes in GUI  | Generating QR codes (only needed when GUI enabled)
+  [zeromq](https://pkgsrc.se/net/zeromq)             | ZMQ notification | Allows generating ZMQ notifications (requires ZMQ version >= 4.0.0)
+  [sqlite3](https://pkgsrc.se/databases/sqlite3)     | SQLite DB        | Wallet storage (only needed when wallet enabled)
+  [python37](https://pkgsrc.se/lang/python37)        | Testing          | Python Interpreter (only needed when running the test suite)
+
+  See [dependencies.md](dependencies.md) for a complete overview.
+
+
+## Preparation
+
+### 1. Install Required Dependencies
+Install the required dependencies the usual way you [install software on NetBSD](https://www.netbsd.org/docs/guide/en/chap-boot.html#chap-boot-pkgsrc) - either with `pkg_add` or `pkgin`. The example commands below use `pkgin` which is usually run as `root` or via `sudo`.
```

Comment:
pkg_add is the default package handler...
 - I believe it is best to use the default package handler.
- when enumerating the build and installation steps - we should not assume that the user has  sudo privileges (if possible).

I believe it is best to reformat this doc with that in mind.

An option is to add pkgin to the initial command.

`
netbsd# pkg_add pkgin autoconf automake libtool pkg-config git gmake boost libevent
`

This assumes the user has sudo privileges, is clearer on the procedure - and eliminates the need to change all the pkgin commands.

Reply:
Thanks for testing! NetBSD's official documentation [recommends](https://www.netbsd.org/docs/guide/en/chap-boot.html#chap-boot-pkgsrc) to use `pkgin` for most binary application tasks. I didn't include it as a dependency because it isn't, the user is free to use `pkg_add` instead of `pkgin`. I also link to documentation on how to install packages on NetBSD so the user can make a choice. Do you think I should explicitly add preparation instructions on installing `pkgin`?

Reply:
Addressed in 1e88f35883ac34914b1a1177244ebd7ff4977ef4. Make it clear that the examples use `pkgin`, link to documentation that shows the tool is the NetBSD recommended tool for installing binary (pre-built) packages